### PR TITLE
Implementing new Magical Effects

### PR DESCRIPTION
**Pull request by Vapok**
Thanks to ploppy for the creativity.

### CODE REVIEW

File: EpicLoot/magiceffects.json [hunk 1]
```diff
@@ -2859,6 +2859,48 @@
         "$mod_epicloot_me_slow_suffix1"
       ]
     },
+    {
+      "Type" : "DoubleMagicShot",
+      "DisplayText" : "$mod_epicloot_me_doublemagicshot",
+	  "Description" : "$mod_epicloot_me_doublemagicshot_desc",
+      "Requirements" : {
+        "AllowedItemTypes" : [
+          "Staff"
```
Comment (RandyKnapp) [inline EpicLoot/magiceffects.json:2868]: I would add SkillType Elemental to this, so it doesn't roll on the protection staff or dead raisser

File: EpicLoot/magiceffects.json [hunk 2]
```diff
@@ -2859,6 +2859,48 @@
...
+      "Type" : "TripleBowShot",
+      "DisplayText" : "$mod_epicloot_me_triplebowshot",
+	  "Description" : "$mod_epicloot_me_triplebowshot_desc",
+      "Requirements" : {
+        "AllowedItemTypes" : [
+          "Bow"
+		]
+      },
+      "ValuesPerRarity" : {
+        "Legendary" : { "MinValue" : 1, "MaxValue" : 1, "Increment" : 1 }
```
Comment (RandyKnapp) [inline EpicLoot/magiceffects.json:2892]: Only legendary? I would allow this on Epic, what do you think?

File: EpicLoot/translations.json [hunk 3]
```diff
@@ -935,7 +935,15 @@
   "mod_epicloot_me_slow_desc": "Slows the attack and movement speed of creatures being hit by +<b><color=yellow>X</color></b>% for 2 seconds. Bosses are immune to this effect.",
   "mod_epicloot_me_slow_prefix1": "Sluggish",
   "mod_epicloot_me_slow_suffix1": "Lethargy",
-
+  "mod_epicloot_me_doublemagicshot": "Double Magic Shot",
+  "mod_epicloot_me_doublemagicshot_desc": "Elemental Magic attacks fire twice, instead of once.",
+  "mod_epicloot_me_doublemagicshot_prefix1": "Elemental Doubling",
+  "mod_epicloot_me_doublemagicshot_suffix1": "of Elemental Doubling",
+  "mod_epicloot_me_triplebowshot": "Triple Bow Shot",
```
Comment (RandyKnapp) [inline EpicLoot/translations.json:942]: The "bow" feels awkward here, maybe just "Triple Shot"?

File: EpicLoot/MagicItemEffects/DoubleMagicShot.cs [hunk 4]
```diff
@@ -0,0 +1,28 @@
+﻿using HarmonyLib;
```
Comment (RandyKnapp) [inline EpicLoot/MagicItemEffects/DoubleMagicShot.cs:1]: Why the split into two files?

**Review by RandyKnapp — CHANGES_REQUESTED** [review]

File: EpicLoot/MagicItemEffects/DoubleMagicShot.cs (hunk 4 above)
Comment (Vapok) [reply EpicLoot/MagicItemEffects/DoubleMagicShot.cs:1]: replied to this via Discord.

Comment (Vapok) [reply EpicLoot/MagicItemEffects/DoubleMagicShot.cs]: Okay this is put back in as one file.

**Review by RandyKnapp — APPROVED** [review]

**Pull request merged**